setup.py

```diff
@@ -53,6 +54,11 @@ def find_version(*file_paths):
     package_dir={"": "src"},
     zip_safe=True,
     include_package_data=True,
+    setup_requires=[
+        # Setuptools 18.0 properly handles Cython extensions.
+        'setuptools>=18.0',
```

Comment:
what's the python version requirement for this?

Reply:
Setuptools 18 was released in 2015. So Python 3.4